source/Hit:Stand.py

```diff
@@ -0,0 +1,17 @@
+def Hit_Stand(self, Player):
+  def_input = input("Would you like to Hit or Stand?  ")
+    while def_input not in ["Hit", "Stand"]:
+        print("Invalid input. Please enter 'Hit' or 'Stand'.")
+        def_input = input("Would you like to Hit or Stand?  ")
+
```

Comment:
7-17 can be simplified by using def_input.upper() you could also use or to combine some of the statement for example
elif def_input =='hit' or def_input =='Hit':